grafana/k6-docs
# Migrate to gatsby-plugin-image

This PR replaces `gatsby-image` with `gatsby-plugin-image`.

**How to test:**
Browse the deployed version pages and make sure that all images look as expected.
